rom/laghos_solver.cpp

```diff
@@ -277,7 +277,9 @@ void LagrangianHydroOperator::SolveVelocity(const Vector &S,
 
         if (use_eqp)
         {
-            rom_op->ForceIntegratorEQP_FOM(rhs);
+			// TODO: this function doesn't have access to the hyperreduction 
+			// sampling type; how should I resolve this?
+            rom_op->ForceIntegratorEQP_FOM(rhs, hyperreduction.SamplingType == eqp_energy);
```

Comment:
In the current scope, I don't have access to the hyperreduction sampling type, which I want to pass as an argument so that the integrator function knows which RHS force vector to form. 

Where should I add the sampling type information so that it's within scope?

Reply:
The `ROM_Operator` class has a private variable `HyperreductionSamplingType hyperreductionSamplingType`. Perhaps you can write an public access function.